doc/guides/maintaining-V8.md

```diff
@@ -144,9 +144,10 @@ includes the following branches<sup>1</sup>:
 </table>
 
 
-The versions of V8 used in Node.js v4.x and v6.x have already been abandoned by
-upstream V8. However, Node.js needs to continue supporting these branches for
-many months (Current branches) or several years (LTS branches).
+The versions of V8 used in Node.js v4.x, v6.x, and 8.x have already been
```

Comment:
should we make this consistent? Either `4.x, 6.x, and 8.x` or `v4.x, v6.x, and v8.x`?

Reply:
Deliberately without the v, because starting with 8, we refer to Node as `Node 8`. But I'm happy to change it if we prefer consistency.

Reply:
ah yea, that's right. I'm indifferent :]